src/main/java/frc/team2412/robot/commands/intake/IntakeDefaultCommand.java

```diff
@@ -7,7 +7,13 @@ public class IntakeDefaultCommand extends IntakeSetCommand {
 	public IntakeDefaultCommand(IntakeSubsystem intakeSubsystem) {
 
 		super(intakeSubsystem, 0);
+	}
+
+	@Override
+	public void execute() {
 		setSpeed(intakeSubsystem.getHoldSpeed());
+		// set motor speed using super initialize?
+		super.initialize();
```

Comment:
Cleaner to directly call `intakeSubsystem.setSpeed(intakeSubsystem.getHoldSpeed())` (especially since we already have access to the subsystem), doesn't matter enough to block merge though.